docs/topics/db/models.txt

```diff
@@ -1415,3 +1415,47 @@ any model field in any ancestor model.
     :doc:`The Models Reference </ref/models/index>`
         Covers all the model related APIs including model fields, related
         objects, and ``QuerySet``.
+
+Organizing models
```

Comment:
This section should go above the seealso which is meant for the bottom of the document.